assets/src/domain/eventEditor/ui/tickets/ticketPriceCalculator/fields/MoneyField/updateInputsSizes.tsx

```diff
@@ -0,0 +1,39 @@
+interface updatedPercentSignInputProps {
+	beforeWidth: number;
+	inputWidth: number;
+	signB4: boolean;
+}
+
+const updatedPercentSignInput = ({ beforeWidth, inputWidth, signB4 }: updatedPercentSignInputProps): void => {
+	document
+		.querySelectorAll('.ee-money-field--with-percent-sign .ee-input__price-field')
+		.forEach((element: HTMLElement) => {
+			if (signB4) {
+				element.style.marginLeft = `${beforeWidth}px`;
+			}
+
+			element.style.width = `${inputWidth}px`;
+		});
+};
```

Comment:
No sure if I understand the need for accessing DOM here.

Reply:
the end result was to avoid these zig zags:

![image](https://user-images.githubusercontent.com/1477453/83167927-68a32c80-a119-11ea-9d06-ccb315c2aac4.png)